test/client/cluster_test.rb

```diff
@@ -62,24 +62,24 @@
   end
 
   it "updates the cluster settings" do
-    @cluster.update_settings transient: { "indices.ttl.interval" => "30m" }
+    @cluster.update_settings transient: { "indices.recovery.max_bytes_per_sec" => "30mb" }
```

Comment:
The TTL setting is not valid in ES 7 so I updated the test to use a different setting.